swift/ios-shoppe-demo/Model/Order.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  Order.swift
+//  ios-shoppe-demo
+//
+//  Created by Harold Davis Jr. on 3/31/20.
```

Comment:
Possible to set the headers by default per https://medium.com/@silmy/how-to-easily-change-your-file-header-text-in-xcode-347fa77d76f4

Reply:
I just found this same guide Ill give this a shot soon and make a club house ticket, thanks! :)